packages/core/fatal-errors/core-fatal-errors-browser-internal/src/index.ts

```diff
@@ -7,5 +7,4 @@
  */
 
 export { FatalErrorsService } from './fatal_errors_service';
-export type { FatalErrorsSetup, FatalErrorsStart } from './fatal_errors_service';
-export type { FatalErrorInfo } from './get_error_info';
+export type { Deps } from './fatal_errors_service';
```

Comment:
NIT: given it's now 'publicly' exported from the package, I'd rename the type to something slightly more explicit, like `FatalErrorsServiceSetupDeps`

Reply:
ya, I was thinking the same.